docs/source/quick_start/supported_types.md

```diff
@@ -20,6 +20,32 @@ provided by the package:
 - [Text](#text-tracking-with-aim)
 - [Figure](#figure-tracking-with-aim)
 
+### Tracking multiple values
+Starting from `v3.14.0` Aim provides ability to track multiple values at once. If the context and epoch are the same,
+the dictionary can be passed and the keys will be treated as names for tracked values. For example the code snippet below
```

Comment:
suggestion:

To track multiple values of different sequences but of the same context, just pass a dictionary of `<name: value>` pairs to the `track` method:

```py
aim_run.track({'accuracy': 98.2, 'loss': 0.001}, context={'subset': 'train'}, step=10, epoch=1)
```

Note: `name` argument should be set to `None` in this case.

further examples ...